person_capture/video_io.py

```
@@ -1479,14 +1479,36 @@ def _q(s: str) -> str:
+    def _maybe_start_initial_pipe(self) -> None:
+        if self._lazy_start_enabled():
+            self._log.info("HDR pipe start deferred until first frame request")
+            return
+        self._start(0)
+
+    def _ensure_started(self) -> bool:
+        if self._proc is not None:
+            return True
+        try:
+            next_idx = max(0, int(getattr(self, "_pos", -1)) + 1)
+        except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)